Users/Glob_Patterns.rst

```diff
@@ -192,13 +182,42 @@ brackets. Parentheses that have no match are ignored as well as
 
 ::
 
-    >>> fnmatch("abc", "a**c")
-    True
     >>> fnmatch("abbc", "a**c")
     True
     >>> fnmatch("a/c", "a**c")
     True
-    >>> fnmatch("a?c", "a**c")
-    True
-    >>> fnmatch("ac", "a**c")
-    True
+
+Using Glob Patterns
+-------------------
+
+Suppose, You want `SpaceConsistencyBear` to perform an analysis on a file
+``first.c``. You should use the command:
+::
+
+    coala --files=src/\first.c --bears=SpaceConsistencyBear --save
+
+.. note::
+
+    If you don't know the fuctions of a bear or how to perform the analysis
+    with a bear, You should go through `Tutorial
+    <http://coala.readthedocs.io/en/latest/Users/Tutorial.html#ignoring
+    -issues>`_ first.
+
+Now, if you want all the ``.c`` files in a specific directory to be analysed,
+you can take help of Glob Patterns.
+::
+
+    coala --files=src/\*.c --bears=SpaceConsistencyBear --save
```

Comment:
rather quote the file glob - the backslashes are distracting and might confuse the user